src/main/java/frc/robot/RobotMap.java

```diff
@@ -1,7 +1,23 @@
 package frc.robot;
 
 public class RobotMap {
+
   public static final class mapDrivetrain {
 
+    public static final int FRONT_LEFT_DRIVE_CAN = 0;
+    public static final int FRONT_LEFT_STEER_CAN = 1;
+    public static final int FRONT_LEFT_ABSOLUTE_ENCODER_CAN = 2;
```

Comment:
just fyi, CANCoder is different device so ID's can actually overlap and this can be 0. It's fine for it to be 2. Just want to make sure you know. CAN bus IDs can only go up to 32

Reply:
👍